low-level/framework/utils/config_reader.py

```diff
@@ -21,17 +21,22 @@
 
 import os
 import sys
-import consul
 import configparser
 import time
 import requests
-import salt.client
 from framework.base.sspl_constants import (component, salt_provisioner_pillar_sls, file_store_config_path,
         SSPL_STORE_TYPE, StoreTypes, salt_uniq_passwd_per_node, COMMON_CONFIGS, SSPL_CONFIGS, CONSUL_PORT,
-        MAX_CONSUL_RETRY, WAIT_BEFORE_RETRY, CONSUL_ERR_STRING, CONSUL_HOST)
+        MAX_CONSUL_RETRY, WAIT_BEFORE_RETRY, CONSUL_ERR_STRING, CONSUL_HOST, PRODUCT_NAME)
 from framework.utils.consulstore import ConsulStore
 from framework.utils.filestore import FileStore
 
+try:

```

Comment:
why this check ? Please add code comments too here

Reply:
LDR_R2: I removed consul and salt dependency for SSPL. 
But for LDR_R1, I keep these as SSPL dependencies for backward compatibility.
If fine, I will add the comment properly.

Reply:
sure, add code comment